### ISSUE TITLE
Remove :voiceover_java_proto_lite from the list of exports in the model library

### ISSUE THREAD

**Issue by fsharpasharp** [oppia/oppia-android#2768]
This task is part of the build system migration from Gradle to Bazel. A prerequesite before working on this issue is that you are able to build Oppia using Bazel. Check out [Oppia Bazel guide](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Guide) for Bazel terminology and more on what Bazel does.

## 1. Remove the export

In 

`/oppia-android/model/BUILD.bazel`

Remove :voiceover_java_proto_lite from the list of exports in the model library

```BUILD
android_library(
    name = "model",
    visibility = ["//visibility:public"],
    exports = [
        ... # <- Remove it from this list.
    ],
)
```

## 2. Add public visibility to the java_lite_proto_library

Add 

`    visibility = ["//visibility:public"],`

to the java_lite_proto_library that has the name voiceover_java_proto_lite.

## 3. Build the app using Bazel and run the tests

Libraries that are indirectly depending on `voiceover_java_proto_lite` will now fail to build. Try to build the entire app with Bazel and add the deps dependency

`//model:voiceover_java_proto_lite,`

to all the libraries that are now failing to build. You will have to look around for BUILD.bazel files that have libraries that require this replaced dependency.

These libraries will already have the //model dependency and since we removed the :voiceover_java_proto_lite from the model library itself, they will need to add the newly available public library directly as a dependency. (Not all libraries with the //model dependency may require the newly available library)

Once the app builds and all tests pass, you're done!

**Comment by Karanjot-singh** [comment]
I would like to work on this issue @prayutsu

**Comment by Karanjot-singh** [comment]
@prayutsu  Please address the following queries as well
1. After completion of task 1 and 2 as mentioned in the issue above, The build was successful. Where am I supposed to add the deps dependency //model:voiceover_java_proto_lite (as per task 3)?

2. How do I lint check my BUILD.bazel file?

3. The following unit Tests failed in the old PR Could you help me with these so that I don't make the same mistakes in the new PR
![image](https://user-images.githubusercontent.com/55680995/112271951-91487400-8ca1-11eb-8a22-384775182afe.png)
![image](https://user-images.githubusercontent.com/55680995/112271970-960d2800-8ca1-11eb-8a0f-68fa9e106992.png)

**Comment by fsharpasharp** [comment]
@Karanjot-singh, if you link a PR, I can answer all these questions for you.

**Comment by Karanjot-singh** [comment]
Thank you @fsharpasharp I am working on a new PR... I will link it soon

**Comment by Karanjot-singh** [comment]
> @Karanjot-singh, if you link a PR, I can answer all these questions for you.

Kindly refer #2999
